### PR TITLE
feat : create ui folder as template

### PR DESCRIPTION
**Pull request by AbdalazizAbdallah**
create a UI folder as a template containing widgets and pages we will design them

create a splash screen as a template
create a Logo widget  as a template
add the whole logo colored and white logo to use them

install packages : 
flutter_svg: ^1.1.4

### CODE REVIEW

**Review by abdkhamis — APPROVED** [review]

**Review by mohammedyousef2 — APPROVED** [review]

File: lib/core/routes/routes_app.dart [hunk 1]
```diff
@@ -2,15 +2,18 @@
...
 class RoutesApp{
 
-  static const String initialPage = '/';
+  static const String initialPage = '/initialPage';
+  static const String splashScreen1 = '/';
 
 
  static   List<GetPage> getPages(){
     return <GetPage>[
       GetPage(name: initialPage, page:()=> const HomePage(),),
```
Comment (fahjan) [inline lib/core/routes/routes_app.dart:15]: did you use initialPage?

File: pubspec.lock [hunk 2]
```diff
@@ -0,0 +1,245 @@
+# Generated by pub
```
Comment (fahjan) [inline pubspec.lock:1]: delete this file please

File: lib/core/routes/routes_app.dart (hunk 1 above)
Comment (AbdalazizAbdallah) [reply lib/core/routes/routes_app.dart:15]: This just template page to team members to put pages in folder and show them when run

File: pubspec.lock (hunk 2 above)
Comment (AbdalazizAbdallah) [reply pubspec.lock:1]: Already ignore it

**Pull request merged**